tests/test_jwt.py

```diff
@@ -288,9 +309,9 @@ def test_decode_no_key_id(token_factory):
 
 
 def test_decode_unknown_alg():
-    headers = json.dumps({u"kid": u"1", u"alg": u"fakealg"})
+    headers = json.dumps({"kid": "1", "alg": "fakealg"})
```

Comment:
Are these changes necessary? We still unofficially support Python2 to a degree.

Reply:
No they aren't but I think they were generated by the formatter suggested in the project.

Reply:
Interesting, thank you

Reply:
Looks like this causes the Python2.7 unit tests to fail. Do you mind manually undoing this?

Reply:
Done!